### PR TITLE
added images for rooms #7

### PR DESCRIPTION
**Pull request by Aswini-D3**
# Description
 We can see the images of the rooms here...so that we can have a visual representation of the room.

Fixes # (issue)
npm run storybook.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by benna100 — COMMENTED** [review]
Why are you renaming the createNewRoomButton component?

File: src/client/components/MessageForm/SendMessageForm.js [hunk 1]
```diff
@@ -15,9 +15,7 @@ function SendMessageForm() {
         'Content-Type': 'application/json',
       },
       body: JSON.stringify(messageInputData),
-    })
-      .then((res) => res.json())
-      .then((messages) => console.log(messages));
+    }).then((res) => res.json());
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.js:18]: This change is not relevant to this story

**Comment by benna100** [comment]
Aswini, this story has been open for 6 days. You have merge conflicts, you are deleting files regarding to send a message and you ar comitting **lots** of unrelated changes. This is not at the level we expect you to be at at this point. I/we have repeatedly told you so many times to have a focus on your PR and the files changed. 

So at this point in time i start getting annoyed when it is not in order. Please fix this

**Comment by Aswini-D3** [comment]
> Aswini, this story has been open for 6 days. You have merge conflicts, you are deleting files regarding to send a message and you ar comitting **lots** of unrelated changes. This is not at the level we expect you to be at at this point. I/we have repeatedly told you so many times to have a focus on your PR and the files changed.
> 
> So at this point in time i start getting annoyed when it is not in order. Please fix this

This one is old one Benjamin..I l already informed you before that I created new branch and made the changes here..."https://github.com/HackYourFuture-CPH/chattie/pull/158#event-3811711824"...

**Comment by Aswini-D3** [comment]
Closing this PR as discussed with Benjamin..Created new branch for this.."https://github.com/HackYourFuture-CPH/chattie/pull/158"...

**Pull request closed without merging**